single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java

```diff
@@ -1,18 +1,21 @@
 package com.github.nhenneaux.resilienthttpclient.singlehostclient;
 
 import java.net.http.HttpRequest;
+import java.util.List;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 
 public class ServerConfiguration {
 
-    private static final int DEFAULT_PORT = -1;
-    private static final String DEFAULT_HEALTH_PATH = "";
-    private static final long DEFAULT_DNS_LOOKUP_REFRESH_PERIOD_IN_SECONDS = TimeUnit.MINUTES.toSeconds(5);
-    private static final long DEFAULT_CONNECTION_HEALTH_CHECK_PERIOD_IN_SECONDS = 30;
-    private static final long DEFAULT_HEALTH_READ_TIMEOUT_IN_MILLISECONDS = TimeUnit.SECONDS.toMillis(5);
-    private static final int DEFAULT_FAILURE_RESPONSE_COUNT_THRESHOLD = -1; // It means no validation by failed response count
+    public static final int DEFAULT_PORT = -1;
+    public static final String DEFAULT_HEALTH_PATH = "";
+    public static final long DEFAULT_DNS_LOOKUP_REFRESH_PERIOD_IN_SECONDS = TimeUnit.MINUTES.toSeconds(5);
+    public static final long DEFAULT_CONNECTION_HEALTH_CHECK_PERIOD_IN_SECONDS = 30;
+    public static final long DEFAULT_HEALTH_READ_TIMEOUT_IN_MILLISECONDS = TimeUnit.SECONDS.toMillis(5);
+    public static final int DEFAULT_FAILURE_RESPONSE_COUNT_THRESHOLD = -1; // It means no validation by failed response count
     public static final Consumer<HttpRequest.Builder> DEFAULT_REQUEST_TRANSFORMER = null;
+    public static final String DEFAULT_PROTOCOL = "https";
+    public static final List<String> PROTOCOLS = List.of("http", "https");
```

Comment:
minor - It can be a Set since always used to verify contains

Reply:
yep, done